test/fetch.js

```diff
@@ -16,3 +16,16 @@ test.serial('relative URLs are passed to fetch unresolved', async t => {
 	t.is(await ky('unicorn', {prefixUrl: '/api/'}).text(), '/api/unicorn');
 	global.fetch = originalFetch;
 });
+
+test.serial('fetch option takes a custom fetch function', async t => {
```

Comment:
This test doesn't need to use `.serial` because, unlike the other test, it doesn't have any global side-effects.